Sources/XCTest/Public/Asynchronous/XCTestCase+NotificationExpectation.swift

```diff
@@ -45,8 +45,7 @@ public extension XCTestCase {
             .default
             .addObserver(forName: Notification.Name(rawValue: notificationName),
                          object: objectToObserve,
-                         queue: nil,
-                         usingBlock: {
+                         queue: nil) {
```

Comment:
I think this is a good change, thanks! But please submit these syntax cleanup changes as a separate pull request.